File: front_end/src/network/system/syslog.js
import {postJson,get} from "../request";

export const syslogHttp = {
  login:function (syslog) {
    let params = {
      url:'/syslog/login',
      data:{

      }
    }
    return postJson(params)
  },
  getEmp:function (logId) {
    let params = {
      url:'/syslog',
      data:{
        'logId':logId
      }
    }
    return get(params)
  },

  queryEmp:function (syslog) {
    let params = {
      url:'/syslog/list',
      data:{
        'logTitle':syslog.logTitle,
        'userId':syslog.userId,
        'logMethod':syslog.logMethod,
        'logContent':syslog.logContent,
        'logIp':syslog.logIp,
        'logUri':syslog.logUri,
        'createTime':syslog.createTime,
      }
    }
    return get(params)
  },
  list:function () {
    let params = {
      url:'/syslog/list'
    }
    return get(params)
  }
}

File: back_end/src/main/java/com/example/entity/request/Syslog.java
package com.example.entity.request;

import com.example.entity.response.EmpResp;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

/**
 * @author: liuleimin
 * @date: 2020/10/21
 * @description:
 */
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Syslog {
    private Integer logId;
    private String logTitle;//日志标题
    private Integer userId;//操作人id
    private String logMethod;//方法
    private String logContent;//操作内容
    private String logIp;//
    private String logUri;//
    private Date createTime;//时间
    private String startDate;
    private String endDate;
}
